diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2f95bc2..4848ed3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,7 @@ endif()
 # Most other platforms have some kind of package manager.
 # FIXME: consider a package manager such as conan/vcpkg instead of vendoring
 if(ANDROID OR MSVC)
-    set(vendored_default TRUE)
+    set(vendored_default FALSE)
 else()
     set(vendored_default FALSE)
 endif()
