diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4211753..470dddc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,7 +36,7 @@ endif()
 # Assume MSVC projects don't have a package manager and need vendored dependencies (by default).
 # Most other platforms have some kind of package manager.
 if(MSVC)
-    set(vendored_default ON)
+    set(vendored_default OFF)
 else()
     set(vendored_default OFF)
 endif()
